util/linux_install.sh

```diff
@@ -95,6 +95,18 @@ elif grep ID /etc/os-release | grep -q gentoo; then
 		echo "Quitting..."
 	fi
 
+elif grep ID /etc/os-release | grep -q sabayon; then
+	sudo equo install \
+		app-arch/unzip \
+		app-arch/zip \
+		app-mobilephone/dfu-util \
+		net-misc/wget \
+		sys-devel/gcc \
+		sys-devel/crossdev dev-embedded/avrdude
+	sudo crossdev -s4 --stable --g =4.9.4 --portage --verbose --target avr
+	echo Done!
+
+
```

Comment:
Might wanna get rid of the double newline here, just for consistency with the rest of the script.